casepro/msgs/models.py

```diff
@@ -452,17 +452,16 @@ def create(cls, org, user, activity, text, contacts, urns, case=None):
 
     def as_json(self):
```

Comment:
Was previously trying to make JSON representation of `Outgoing` match that of `Message` but these models continue to diverge so it's not worth it.